akka-bench-jmh-typed/src/main/scala/akka/actor/typed/TypedBenchmarkActors.scala

```diff
@@ -19,10 +19,9 @@ object TypedBenchmarkActors {
   // we pass the respondTo actor ref into the behavior
   final case object Message
 
-  private def echoBehavior(respondTo: ActorRef[Message.type]): Behavior[Message.type] = Behaviors.receive {
-    (ctx, msg) =>
-      respondTo ! Message
-      Behaviors.same
+  private def echoBehavior(respondTo: ActorRef[Message.type]): Behavior[Message.type] = Behaviors.receiveMessage { _ =>
```

Comment:
since this is a benchmark I think the intention was to use the full `receive`

Reply:
I'll revert